robyn/__init__.py

```diff
@@ -80,11 +90,30 @@ def _add_route(
         """
         if auth_required:
             self.middleware_router.add_auth_middleware(endpoint)(handler)
-
+        print("self get dict from init to router.py", self.dependencies.dependency_map)
         return self.router.add_route(
-            route_type, endpoint, handler, is_const, self.exception_handler
+            route_type,
+            endpoint,
+            handler,
+            is_const,
+            self.dependencies.dependency_map,
+            self.exception_handler,
         )
 
+    def inject(self, route=None, http_method=None, **kwargs: dict):
+        # the http_method param is unused??
+        # @Darren
```

Comment:
@DarrenSeng

Reply:
http_method as a param was shown in one of the example cases in one of your comments on the issue page, but I'm still not sure how to make use of it, so I'll probably delete that param